packages/admin-ui/src/calendar/calendar-cell-button.ts

```diff
@@ -0,0 +1,104 @@
+import { useRef, useEffect, useCallback } from 'react'
+import { Clickable } from 'reakit'
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+import { callAllHandlers, ensureFocus } from '@vtex/admin-ui-util'
+import { useForkRef } from '@vtex/admin-ui-hooks'
+import { isSameDay } from 'date-fns'
+
+import { useDateFormatter } from '../i18n'
+import * as style from './calendar.style'
+import type { CalendarStateReturn } from './calendar-state'
+
+export const CalendarCellButton = createComponent<
+  typeof Clickable,
+  CalendarCellButtonOptions
+>((props) => {
+  const {
+    date,
+    state: {
+      isDisabled: isDisabledOption,
+      month,
+      isInvalidDateRange,
+      dateValue,
+      focusedDate,
+      isFocused: isFocusedOption,
+      selectDate,
+      setFocusedDate,
+    },
+    disabled: htmlDisabled,
+    onClick: htmlOnClick,
+    onFocus: htmlOnFocus,
+    ref: htmlRef,
+    ...htmlProps
+  } = props
+
+  const ref = useRef<HTMLElement>(null)
+  const isCurrentMonth = date.getMonth() === month
+  const isDisabled =
+    isDisabledOption || !isCurrentMonth || isInvalidDateRange(date)
```

Comment:
Date picker is only for the current month? 🤔

Reply:
Not really. Condition is for days since we have dates for the current, past, and next month. If you're not in the current month, the day is disabled.

Reply:
Ah ok I get it